src/NHibernate.Test/NHibernate.Test.csproj

```diff
@@ -58,7 +58,9 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)'=='net461'">
     <Reference Include="System.Configuration" />
+    <Reference Include="System.Runtime" />
     <Reference Include="System.Transactions" />
+    <Reference Include="System.Threading.Tasks" />
```

Comment:
There is System.Runtime and System.Threading.Tasks that come from some ADO.NET provider (I think this is Npgsql) which do not match the ones from the `/Facades` folder, so reference them explicitly.